### ISSUE TITLE
Screenshots as attachment

### ISSUE THREAD

**Issue by bitsandfoxes** [getsentry/sentry-unreal#567]
Let's look into attaching - or opting into where applicable - attaching screenshots to events captured by the SDK.
- [x] Windows
- [x] Linux
- [x] macOS
- [ ] Android
- [x] iOS

**Comment by bruno-garcia** [comment]
I see #582 was merged last month, is this capability mentioned on the docs? I couldn't find it

**Comment by tustanivsky** [comment]
> I see [#582](https://github.com/getsentry/sentry-unreal/pull/582) was merged last month, is this capability mentioned on the docs? I couldn't find it

You're right, thanks for noticing! This feature hasn't been reflected in the docs yet since screenshot capturing is limited to work only on Win/Linux but I'll make sure to add it there with a corresponding notice about currently supported platforms.

**Comment by bruno-garcia** [comment]
If you end up adding docs, please don't forget to link back to this issue so we can trace related work through

**Comment by bruno-garcia** [comment]
Blocked on:
* https://github.com/getsentry/sentry-unreal/issues/430

**Comment by bruno-garcia** [comment]
Also note that 2 of the the missing checkboxes on this issue have support OOTB:

* https://docs.sentry.io/platforms/apple/guides/ios/enriching-events/screenshots/
* https://docs.sentry.io/platforms/android/enriching-events/screenshots/

We just need to pass that flag down
